crates/types/protobuf/restate/cluster.proto

```diff
@@ -65,6 +65,7 @@ message PartitionProcessorStatus {
   uint64 num_skipped_records = 8;
   ReplayStatus replay_status = 9;
   optional restate.common.Lsn last_persisted_log_lsn = 10;
+  optional restate.common.Lsn last_archived_log_lsn = 12;
```

Comment:
Could those be unified? I am not sure how much sense `last_persisted_log_lsn` makes in the presence of snapshots.

Reply:
I wasn't sure initially, but am increasingly convinced we only need one attribute to represent the "durably persisted" LSN.

The counter-argument is that it's better to be explicit than implicit; if the meaning of "safely persisted to external storage" changes based on config, we should surface this very clearly to the operator, as a misunderstanding could have huge consequences.

Let me think about this a bit more; I've crated https://github.com/restatedev/restate/issues/2306 so we don't drop this.